pages/spicedb/modeling/validation-testing-debugging.mdx

```diff
@@ -85,6 +88,18 @@ project:docs#admin:
   - "[user:rauchg] is <platform:vercel#admin>"
 ```
 
+## Check Tracing
+
+SpiceDB supports tracing of check requests to view the path(s) taken to compute the result, as well as timing information.
+
+### Versions older than v1.31.0
+
+Request tracing information via a header and the information is found in the response footer as JSON.
+
+### v1.31.0 or later
+
+Request tracing information by setting `with_tracing: true` in the request message and the information is found in the response message.
+
```

Comment:
I think it might be a better structure to the page if this just documents the v1.31 behavior with a warning about using the header for older versions rather than having two additional layers of nesting headings.